tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1683,6 +1771,93 @@ def test_copy_pds_member_with_system_symbol(ansible_zos_module,):
         hosts.all.zos_data_set(name=dest, state="absent")
 
 
+@pytest.mark.pdse
+def test_copy_pds_loadlib_member_to_pds_loadlib_member(ansible_zos_module,):
+    hosts = ansible_zos_module
+    # The volume for this dataset should use a system symbol.
+    # This dataset and member should be available on any z/OS system.
+    src = "USER.LOAD.SRC"
+    dest = "USER.LOAD.DEST"
+    cobol_pds = "USER.COBOL.SRC"
+    try:
+        hosts.all.zos_data_set(
+            name=src,
+            state="present",
+            type="pdse",
+            record_format="U",
+            record_length=0,
+            block_size=32760,
+            space_primary=40,
```

Comment:
I know this is a result of the playbook I provided and no idea why i had the space_primary so large, I don't think any of these will really consume 40 MB, although not a show stopper right now , eventually we might want to be a bit more conservative with space. This is in all three data sets but ill comment here only.

Reply:
Thanks for the recommendation. Reduced it to 2MB per dataset.